projects/manager/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "../../.eslintrc.json",
+    "ignorePatterns": ["!**/*"],
+    "overrides": [
+        {
+            "files": ["*.ts"],
+            "parserOptions": {
+                "project": ["projects/manager/tsconfig.app.json", "projects/manager/tsconfig.spec.json"],
+                "createDefaultProgram": true
+            },
+            "rules": {}
+        },
+        {
+            "files": ["*.html"],
+            "rules": {}
+        }
+    ]
+}
```

Comment:
We should display the result descriptor.
Currently, we cannot see the columns and their names and units.

Reply:
and the colorizer!

Reply:
Awesome, there is one glitch I guess:

![grafik](https://github.com/geo-engine/geoengine-ui/assets/4947061/1af4ef95-3d83-434d-a3cc-93fb26a79317)

"ndvi_flipped_axis"

Reply:
no that's the name of the band